static/src/js/actions/errors.js

```diff
@@ -3,11 +3,20 @@ import uuidv4 from 'uuid/v4'
 import { REMOVE_ERROR, ADD_ERROR } from '../constants/actionTypes'
 import LoggerRequest from '../util/request/loggerRequest'
 import { parseError } from '../../../../sharedUtils/parseError'
+import { displayNotificationType } from '../constants/enums'
+import { pushErrorNotification } from './notifications'
 
-export const addError = payload => ({
-  type: ADD_ERROR,
-  payload
-})
+export const addError = payload => (dispatch) => {
```

Comment:
As mentioned in the ErrorBannerContainer, we should prevent adding errors to Redux if we're not requesting a banner, an example update to this method would be: 

```
export const addError = payload => (dispatch) => {
  // Default the notificationType to none
  const { notificationType = 'none' } = payload

  if (notificationType === 'banner') {
    dispatch({
      type: ADD_ERROR,
      payload
    })
  } else if (notificationType === 'toast') {
    const { message } = payload

    // Disable autoDismiss for errors
    addToast(message, {
      appearance: 'error',
      autoDismiss: false
    })
  }
}
```